# ASM_Poster
## American Society of Microbiology 2025 Poster References

- **metaSPAdes (metagenomic assembler)**  
Nurk S, Meleshko D, Korobeynikov A, Pevzner PA. *metaSPAdes: a new versatile metagenomic assembler.* Genome Res. 2017 May;27(5):824–834. doi:10.1101/gr.213959.116  

- **MetaBAT 2** (Kang *et al.*, 2019)  
  Kang D.D., Li F., Kirton E., Thomas A., Egan R., An H., Wang Z. *MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies.* PeerJ 7:e7359. doi:10.7717/peerj.7359 

- **BASALT** (Qiu *et al.*, 2024)  
  Qiu Z., Yuan L., Lian C.-A., *et al.* *BASALT refines binning from metagenomic data and increases resolution of genome-resolved metagenomic analysis.* Nature Communications 15:2179. doi:10.1038/s41467-024-46539-7 

- **MaxBin 2** (Wu *et al.*, 2016)  
  Wu Y.-W., Simmons B.A., Singer S.W. *MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets.* Bioinformatics 32(4): 605–607. doi:10.1093/bioinformatics/btv638 

- **CheckM2** (Chklovski *et al.*, 2023)  
  Chklovski A., Parks D.H., Woodcroft B.J., *et al.* *CheckM2: a rapid, scalable and accurate tool for assessing microbial genome quality using machine learning.* Nature Methods 20: 1203–1212. doi:10.1038/s41592-023-01940-w

- **dRep** (Olm *et al.*, 2017)  
  Olm M.R., Brown C.T., Brooks B., Banfield J.F. *dRep: a tool for fast and accurate genomic comparisons…* ISME Journal 11: 2864–2872. doi:10.1038/ismej.2017.126

- **GTDB‑Tk** (Chaumeil *et al.*, 2020)  
  Chaumeil P.-A., Mussig A.J., Hugenholtz P., Parks D.H. *GTDB‑Tk: a toolkit to classify genomes with the Genome Taxonomy Database.* Bioinformatics 36(6): 1925–1927. doi:10.1093/bioinformatics/btz848 

- **CoverM** (Aroney *et al.*, 2025)  
  Aroney S.T.N., Newell R.J.P., Nissen J.N., Camargo A.P., Tyson G.W., Woodcroft B.J. *CoverM: read alignment statistics for metagenomics.* Bioinformatics 41(4):… btaf147. doi:10.1093/bioinformatics/btaf147

- **Prodigal** (Hyatt *et al.*, 2010)  
  Hyatt D., Chen G.-L., LoCascio P.F., *et al.* *Prodigal: prokaryotic gene recognition and translation initiation site identification.* BMC Bioinformatics 11:119. doi:10.1186/1471-2105-11-119

- **WGCNA** (Langfelder & Horvath, 2008)  
  Langfelder P., Horvath S. *WGCNA: an R package for weighted correlation network analysis.* BMC Bioinformatics 9:559. doi:10.1186/1471-2105-9-559

- **METABOLIC-G** (Zhou *et al.*, 2022)  
  Zhou Z., Tran P.Q., Breister A.M., *et al.* *METABOLIC: high-throughput profiling of microbial genomes for functional traits, metabolism, biogeochemistry, and community-scale functional networks.* Microbiome 10:33. doi:10.1186/s40168-021-01213-8

- **SPAdes (core algorithm)** (Bankevich *et al.*, 2012)  
  Bankevich A., Nurk S., Antipov D., *et al.* *SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.* Journal of Computational Biology 19(5): 455–477


